src/i18n/en.json

```diff
@@ -45,7 +45,8 @@
     },
 
     "common": {
-      "browser-not-supported": "It seems your browser is not supported. Some of the features may not work"
+      "browser-not-supported": "It seems your browser is not supported. Some of the features may not work",
+      "button-text-more": "More"
```

Comment:
`more-btn`

Reply:
@LemonJenkins  still not resolved, rename to `more-btn`

Reply:
ok